### DISCUSSION TITLE
Front Camera topic not publishing anything

### DISCUSSION THREAD

**Discussion by rj-indro** [abizovnuralem/go2_ros2_sdk#95] [General]
Hello,  I am trying to setup the go2 to use Cyclonnedds and pipe all the ros topics over rosbridge.

I've managed to get most such as the pointclouds and urdf however I can't get any ros topic data from the camera after subscribing to it, either from over rosbridge or locally.  Any idea what could be causing this?

Thanks!

**Comment by abizovnuralem** [comment]
Hello, for Cyclonnedds you need to read official unitree docs on how to use web camera stream with it

**Comment by kopebryant** [comment]
Hi, did you get it running again?
